React/Base/RCTConvert.m

```diff
@@ -882,7 +882,7 @@ + (UIImage *)UIImage:(id)json
   NSString *scheme = URL.scheme.lowercaseString;
   if ([scheme isEqualToString:@"file"]) {
     NSString *assetName = RCTBundlePathForURL(URL);
-    image = [UIImage imageNamed:assetName];
+    image = [UIImage imageNamed:[NSString stringWithFormat:@"%@", assetName]];
```

Comment:
Totally agree, thanks @ide and @nicklockwood ...first time submitting PR to this repo, why did I not pass the The Travis CI?